snmp/datadog_checks/snmp/data/profiles/meraki-cloud-controller.yaml

```diff
@@ -0,0 +1,107 @@
+# Profile for Meraki Cloud devices
+#
+# You need the MIB compiled to get the data. You can run the following command to get it:
+# $ /opt/datadog-agent/embedded/bin/python /opt/datadog-agent/embedded/bin/mibdump.py  --destination-directory=/opt/datadog-agent/embedded/lib/python2.7/site-packages/pysnmp_mibs MERAKI-CLOUD-CONTROLLER-MIB.mib
+#
+sysobjectid: 1.3.6.1.4.1.29671.*
+metrics:
+  # Meaki cloud devices
+  - MIB: MERAKI-CLOUD-CONTROLLER-MIB
+    symbol: devStatus # 0 or 1 values
+    forced_type: gauge
+    metrics_tags:
+      - tag: device
+        column: devName
+      - tag: product
+        column: devProductCode
+      - tag: network
+        column: devNetworkName
+      - tag: vlan
+        column: vlanNetworkName
+  - MIB: MERAKI-CLOUD-CONTROLLER-MIB
+    symbol: devClientCount
```

Comment:
It's in the same table as devStatus, you want to add a `table` header and put both symbols in there.